lib/cogctl/actions/relays.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Cogctl.Actions.Relays do
+  use Cogctl.Action, "relays"
+  alias Cogctl.Table
+
+  def option_spec do
+    []
+  end
+
+  def run(_options, _args, _config, endpoint),
+    do: with_authentication(endpoint, &do_list/1)
+
+  defp do_list(endpoint) do
+    case CogApi.HTTP.Client.relay_index(endpoint) do
+      {:ok, relays} ->
+        relay_rows = Enum.map(relays, &format_table(&1))
+        render([["NAME", "STATUS", "CREATED", "ID"]] ++ relay_rows, true)
+      {:error, error} ->
+        display_error(error)
+    end
+  end
+
+  def get_details(relay) do
```

Comment:
I use this function (and render and get_status) in the other relay commands as well. I thought about putting them in action_util.ex, but they are relay focused, so I put them here and then import them into the relay commands that use them.